presto-native-execution/presto_cpp/main/types/tests/PlanConverterTest.cpp

```diff
@@ -62,6 +66,24 @@ std::shared_ptr<const core::PlanNode> assertToVeloxQueryPlan(
           prestoPlan, nullptr, "20201107_130540_00011_wrpkw.1.2.3")
       .planNode;
 }
+
+std::shared_ptr<const core::PlanNode> assertToBatchVeloxQueryPlan(
+    const std::string& fileName,
+    const std::shared_ptr<operators::ShuffleInterface>& writerShuffle) {
+  std::string fragment = slurp(getDataPath(fileName));
```

Comment:
Hard-coding JSON representation of the query fragments in tests is brittle as the representation changes frequently.

Reply:
Hi Masha, do you have other recommended ways of this test?